src/components/ShareTrajectoryModal/EmbedSnippetPanel.tsx

```diff
@@ -32,8 +32,8 @@ const EmbedSnippetPanel = ({ startTime }: EmbedSnippetPanelProps) => {
     const [showEmbedSettingsPanel, setShowEmbedSettingsPanel] =
         React.useState(false);
 
-    const url = `https://simularium.allencell.org/embed?trajFileName=${trajectory}&t=${startTime}`;
-    const embedSnippet = `<iframe height="${width}" width="${height}" src="${url}" title="Simularium" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>`;
+    const url = `http://${location.host}/embed?trajFileName=${trajectory}&t=${startTime}}`;
```

Comment:
origin is probably better here so you don' have to change the protocol

Reply:
I would also add `frameBorder="0"` to get rid of the auto iframe border

Reply:
ok yes to frame border, and for now yes to origin, but just to check my understanding, we don't want to merge either version of using `location` we want to point directly to `simularium.allencell.org` once we get done with review

Reply:
we do want to merge it. when it's on the website, location will be simularium.allencell.org. or staging.simularium.allencell.org which will be more consistent for testing